### PR TITLE
Bug 1970604:  Add IDP menu items are not translated

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @rhamilto 
cc: @rebeccaalpert 

Screen Shot 2021-06-21 at 11 20 32 AM

### CODE REVIEW

File: frontend/public/components/cluster-settings/oauth.tsx [hunk 1]
```diff
@@ -73,17 +64,43 @@ const OAuthDetails: React.FC<OAuthDetailsProps> = ({ obj }: { obj: OAuthKind })
...
+        return type;
+    }
+  };
+
+  const IDPDropdownItems = Object.entries(addIDPItems).map((idp) => {
+    const [key, value] = idp;
+
     return (
       <DropdownItem
-        key={`idp-${addIDPItems[idp]}`}
```
Comment (cyril-ui-developer) [inline frontend/public/components/cluster-settings/oauth.tsx:80]: @rhamilto @rebeccaalpert I am not sure why we were assigning `addIDPItems` object value here. I think it should be the key.

File: frontend/public/components/cluster-settings/oauth.tsx [hunk 2]
```diff
@@ -47,15 +47,6 @@ const IdentityProviders: React.FC<IdentityProvidersProps> = ({ identityProviders
...
 
-// t('public~Basic Authentication')
-// t('public~GitHub')
-// t('public~GitLab')
-// t('public~Google')
-// t('public~HTPasswd')
-// t('public~Keystone')
-// t('public~LDAP')
-// t('public~OpenID Connect')
-// t('public~Request Header')
```
Comment (cyril-ui-developer) [inline frontend/public/components/cluster-settings/oauth.tsx:58]: @rhamilto @rebeccaalpert i don't think we need the above any more.

**Comment by rhamilto** [comment]
LGTM but will wait for @rebeccaalpert to give the final approval as the resident i18n expert.  :)

File: frontend/public/components/cluster-settings/oauth.tsx (hunk 2 above)
Comment (rebeccaalpert) [reply frontend/public/components/cluster-settings/oauth.tsx:58]: You're totally right; we don't need these anymore.

**Review by rebeccaalpert — APPROVED** [review]
Looks great to me. :) Thanks for fixing this.

**Pull request merged**